List get-started content under guides

simon04 opened the pull request:
Related: #743, #749.

Supersedes #773 (wrong source branch).

bebraw commented:
It might be a good idea to write redirects for the pages. The syntax is like this:

```javascript
    redirects: {
      'get-started': '/guides/get-started'
    }
```

That should go below the path definition next to `path()` etc. Consider building only that portion of the site when testing the build (comment out everything else).

simon04 commented:
I added the redirects, and extracted the syntax from [here](https://github.com/antwarjs/antwar/blob/619fa298abc8db0771104623d18e68610e0d49eb/smoketest/antwar.config.js#L51-L55).
